exercises/leap/source/leap/LeapTest.ceylon

```diff
@@ -0,0 +1,10 @@
+import ceylon.test { ... }
+
+shared {[Integer, Boolean]*} cases =>
+  {[2015, false], [2016, true], [2100, false], [2000, true]};
+
+test
+parameters(`value cases`)
+shared void testLeapYear(Integer year, Boolean isLeap) {
+  assert(leapYear(year) == isLeap);
```

Comment:
try `assertEquals` instead of this plain `assert(a == b)`, see if it gives richer info.